core/primitives/src/version.rs

```diff
@@ -166,7 +168,7 @@ const STABLE_PROTOCOL_VERSION: ProtocolVersion = 60;
 /// Largest protocol version supported by the current binary.
 pub const PROTOCOL_VERSION: ProtocolVersion = if cfg!(feature = "nightly_protocol") {
     // On nightly, pick big enough version to support all features.
-    135
+    136
```

Comment:
Note: `tests::simple_chain::build_chain` (with nightly features enabled) fails in CI because this changes the protocol config. You need to update the snapshot. It's a bit annoying but it also helps to prevent some types of accidental protocol updates.

Reply:
Thanks for the pointer, I ran
```
cargo insta test --accept -p near-chain --features nightly -- tests::simple_chain::build_chain
```
to fix this.